src/core/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import useStyles from './styles';
+import { TabBaseProps } from './base';
+import Tab from '@material-ui/core/Tab';
+import TabList from '@material-ui/lab/TabList';
+import AppBar from '@material-ui/core/AppBar';
+import TabPanel from '@material-ui/lab/TabPanel';
+import TabContext from '@material-ui/lab/TabContext';
+
+interface TabsProps extends TabBaseProps {
+  label: string[];
+  content: any[];
+}
+
+const Tabs: React.FC<TabsProps> = ({ label, content }) => {
+  const classes = useStyles();
+  const [value, setValue] = React.useState('0');
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: string) => {
```

Comment:
we have usecases of handling this tab changes with external state management solutions like redux...so the value and onChange values needs to come as props

Reply:
@arkajyotiMukherjee We must  match the content and label through TabPanel and Tab values. Do I need to pass one props as a value?

Reply:
yes value and onChange will be props
```
value={value}
onChange={onChange}
```